wikipedia-dark.user.css

```diff
@@ -384,11 +413,297 @@ regexp("https?:\/\/.*wik(i|t).*(org|jp).*") {
   .ve-ui-mwAceEditorWidget .ace_editor {
     border-color: var(--cm-6);
   }
+	/*Blyad's Fixes*/
+	.central-featured-lang :hover, .central-featured-lang :hover strong::before, .other-project-link:hover, .central-featured-lang :hover small::before, .central-featured-lang :hover bdi::before, .central-featured-lang :hover span::before {
+		background-color: #0000;
+		position: relative;
+	}
+	.central-featured-lang :hover::before, .other-project-link:hover::before {
+		content: '';
+		position: absolute;
+		width: 100%;
+		height: 100%;
+		left: 0;
+		top: 0;
+		margin: 0;
+		background-color: var(--blue-3);
+		opacity: .54;
```

Comment:
Why is it needed to override all these properties?

Reply:
Nothing here is overrided, it is pseudo-element to allow me to make a nice transparent background without adding new variable,